src/main/groovy/pl/allegro/tech/build/axion/release/domain/VersionResolver.groovy

```diff
@@ -24,16 +24,33 @@ class VersionResolver {
     private final ScmRepository repository
 
     private final VersionSorter sorter
+    private ScmPosition latestChangePosition
+    private String projectRootRelativePath
+    private List<String> foldersToExcludeWhenResolvingLatestRelevantRootProjectCommit
 
     VersionResolver(ScmRepository repository) {
+        this(repository, null, Collections.emptyList())
```

Comment:
if we need only one constructor - go ahead and remove this one :)

Reply:
Amended.